### PR TITLE
feat: show multisig only for available networks

### PR DESCRIPTION
**Pull request by stepanLav**

### CODE REVIEW

File: metadataCreator/src/utils/apiUtils.ts [hunk 1]
```diff
@@ -1,19 +1,39 @@
+import axios from 'axios';
+import yaml from 'js-yaml';
 import { Chain } from "../models/Chain";
 import { MultisigVersionStorage } from "../models/MultisigVersionStorage";
 
+const repoUrl = 'https://api.github.com/repos/novasamatech/subquery-proxy/contents/';
+
+async function fetchYamlFiles(): Promise<any[]> {
```
Comment (Asmadek) [inline metadataCreator/src/utils/apiUtils.ts:8]: Maybe use more meaningful name? `fetchSubqueryConfigs` for example

File: metadataCreator/src/utils/apiUtils.ts [hunk 2]
```diff
@@ -1,19 +1,39 @@
+import axios from 'axios';
+import yaml from 'js-yaml';
 import { Chain } from "../models/Chain";
 import { MultisigVersionStorage } from "../models/MultisigVersionStorage";
 
+const repoUrl = 'https://api.github.com/repos/novasamatech/subquery-proxy/contents/';
```
Comment (Asmadek) [inline metadataCreator/src/utils/apiUtils.ts:6]: Should it be `subquery-proxy` not multisig?

**Review by Asmadek — APPROVED** [review]

**Comment by pgolovkin** [comment]
@stepanLav this PR should not be merged until new multisig are implemented and released.

**Comment by stepanLav** [comment]
> @stepanLav this PR should not be merged until new multisig are implemented and released.

ok, change it to the draft

File: metadataCreator/src/utils/apiUtils.ts (hunk 2 above)
Comment (Tbaut) [reply metadataCreator/src/utils/apiUtils.ts:6]: it certainly should, this will be misleading for many otherwise. Should I create an issue?

**Review by tuul-wq — APPROVED** [review]

**Review by Tbaut — APPROVED** [review]

**Review by pgolovkin — APPROVED** [review]

**Pull request merged**